### ISSUE TITLE
UDC Bot should only warn users once every X when they try to @everyone

### ISSUE THREAD

**Issue by Pierre-Demessence** [Unity-Developer-Community/UDC-Bot#106]
When a user try to `@everyone`, they get a warning message:
![image](https://user-images.githubusercontent.com/1756398/134774870-60d4f2e7-7d81-4aa9-947d-e5eda6c03720.png)

When we get bot spams, they often `@everyone` and thus the bot spams the warning.
To prevent that, it would be nice to have the bot keep track (in memory, no need for yet another file) of warnings recently sent per user and only send another warning if the last one was X minutes/hours/days ago. (Configurable in setting file)

So basically a dictionary of `user => time` that we check before warning again.
